### PR TITLE
Print pretty messages in `quint verify`

### PR DESCRIPTION
**Pull request by thpani**
Addresses (1) of #1121:

Print pretty messages in `quint verify`, similar to `quint run`

- [x] Tests added for any new code
- [ ] ~Documentation added for any new functionality~
- [ ] ~Entries added to the respective `CHANGELOG.md` for any new functionality~
- [ ] ~Feature table on [`README.md`](../README.md#roadmap) updated for any listed functionality~

### CODE REVIEW

**Comment by thpani** [comment]
> We should try to reuse these `console.log` statements that I believe are now repeated in simulation/verification.

Also `test` 😄

I tried but it's difficult. E.g., for simulation it says something about `--max-steps` and `--max-samples`, but verification has only `--max-steps`.

**Review by konnov — APPROVED** [review]
Looks great! Thanks for taking care of this!

**Comment by konnov** [comment]
I am going to merge this one, as it would be extremely nice to have this feature in the upcoming release.

File: quint/src/cli.ts [hunk 1]
```diff
@@ -238,7 +238,12 @@ const verifyCmd = {
       .option('apalache-config', {
         desc: 'Filename of the additional Apalache configuration (in the HOCON format, a superset of JSON)',
         type: 'string',
-      }),
+      })
+      .option('verbosity', {
+        desc: 'control how much output is produced (0 to 5)',
+        type: 'number',
+      })
+      .default('verbosity', verbosity.defaultLevel),
```
Comment (shonfeder) [inline quint/src/cli.ts:246]: The default can be supplied in the option dict, as on lines 221 and 226 above:

```suggestion
      .option('verbosity', {
        desc: 'control how much output is produced (0 to 5)',
        type: 'number',
        defualt: verbosity.defaultLevel
      })
```

**Review by shonfeder — APPROVED** [review]

**Comment by konnov** [comment]
It looks like we are getting GH errors when checking for a fresh version of Apalache:

>     error: Error listing the Apalache releases: HttpError: API rate limit exceeded for 207.254.48.161. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)

**Comment by shonfeder** [comment]
@konnov I'm tracking that in #1124

**Pull request merged**